frontend/public/components/modals/_modals.scss

```diff
@@ -51,6 +51,21 @@
   .input-group-addon {
     background-color: rgba(227, 227, 227, 0.5);
   }
+
+  .pf-c-alert {
+    margin-top: var(--pf-global--spacer--md);
+  }
+
+  // fixes title misalignment with icon
+  .pf-c-alert__title {
+    display: flex;
+    align-items: center;
+  }
```

Comment:
We definitely don't want to be overriding the `pf-c-alert__title` class generally. We have other modals that have an icon. Can we reuse the style from those?

Reply:
Looking into this.